v2/internal/attacktechniques/gcp/exfiltration/share-compute-snapshot/main.go

```diff
@@ -0,0 +1,172 @@
+package gcp
+
+import (
+	compute "cloud.google.com/go/compute/apiv1"
+	"context"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"log"
+
+	_ "embed"
+	"fmt"
+	"github.com/datadog/stratus-red-team/v2/internal/providers"
+	gcp_utils "github.com/datadog/stratus-red-team/v2/internal/utils/gcp"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
+)
+
+//go:embed main.tf
+var tf []byte
+
+const codeBlock = "```"
+const AttackTechniqueId = "gcp.exfiltration.share-compute-snapshot"
```

Comment:
should probably use it in the `ID: ` definition below if you're extacting it as a constant

Reply:
I see you just fixed it, thanks!